### ISSUE TITLE
Aleatoric and epistemic combined in run_cubic_tests.py for ensemble and dropout

### ISSUE THREAD

**Issue by 25benjaminli** [aamini/evidential-deep-learning#21]
Hi, I believe in `neurips2020/run_cubic_tests.py` what is actually being plotted in functions like `plot_ensemble` and `plot_dropout` are total uncertainty rather than the epistemic uncertainty. Because the aleatoric uncertainty is small relative to epistemic the difference is not noticeable but problematic nonetheless. What is plotted for the evidential models in `plot_ng` is solely epistemic uncertainty.  

https://github.com/aamini/evidential-deep-learning/blob/d1d8e395fb083308d14fa92c5ce766e97b2a066a/neurips2020/run_cubic_tests.py#L58C1-L66C1

The following is what I get for the ensemble after doing the proper separation by redefining the metrics to
```
epistemic = tf.math.reduce_std(mus, axis=0)
aleatoric = tf.reduce_mean(sigmas, axis=0)
```
and adding a custom plotting function to view the separation of epistemic vs. aleatoric

<img width="1356" height="394" alt="Image" src="https://github.com/user-attachments/assets/2b4d6969-bbd2-461f-a084-0ddc8c5a5ccd" />
